libraries/rush-lib/src/cli/actions/InitAction.ts

```diff
@@ -178,6 +185,12 @@ export class InitAction extends BaseConfiglessRushAction {
       'rush.json'
     ];
 
+    const experimentalTemplateFilePaths: string[] = ['subspaces.json'];
```

Comment:
Put this filename in the constants file and grab it from there.

Reply:
Changed